routes/api/contacts.js

```diff
@@ -1,25 +1,25 @@
-const express = require('express')
+import express from "express";
 
-const router = express.Router()
+import { showContacts } from "../../controllers/contacts/showContacts.js";
```

Comment:
Nie widzę tego importu :) Chyba nie ma wszystkich plików?

Reply:
Tak, przy commit nie zwróciłam uwagi na fakt że folder controllers się nie dodał. Już zostało to poprawione.